### PR TITLE
Alden - Lab 8 complete with updates, Labs 3 & 6 v2

### PR DESCRIPTION
**Pull request by Alden-Addor**
Finished right at the end of class today!!

### CODE REVIEW

File: code/alden/python/lab08.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
+import re
```
Comment (brucestull) [inline code/alden/python/lab08.py:1]: Needs Work:
* REPL should keep looping until the user decides to end the loops.
    * User should be able to do the four CRUD without having to restart the python program.
* See me if you have any questions.

**Review by brucestull — CHANGES_REQUESTED** [review]
[Lab 8: Contact List](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/08%20Contact%20List.md)

Needs Work:
* User should be able to repeatedly do any of the four CRUD operations and then have some option to 'exit' or 'quit' when they are finished.

**Comment by Alden-Addor** [comment]
Updated to use a REPL to keep 'user' going until they wish to leave.  Sorry, I definitely missed that in the instructions.

**Comment by brucestull** [comment]
Much appreciation for updating Pull Request Title. It makes tracking lab reviews more effective and effictient.

File: code/alden/python/lab08.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    else:
+        cont_update.append(list(sub.values()))
+
+lines =[]
+for i in range(0, len(cont_update)):            # Takes it back to a list of stings as it was when first imported for the .csv doc.
+    lines.append(','.join(cont_update[i]))
+    
+lines = '\n'.join(lines)    # Puts everything back into a string and separates them onto different lines to be ready to put back into the .csv file.
+
+with open('contacts.csv', 'w') as phone_book_file:  # Writes over the .csv file, updating any changes.
```
Comment (brucestull) [inline code/alden/python/lab08.py:91]: Logic above works as necesary so the final write overwrites the file and we can access that same file again.

Great job!

**Review by brucestull — APPROVED** [review]
[Lab 8: Contact List](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/08%20Contact%20List.md)

**Completed**

Great Job!
* Good use of flavor text.

**Pull request merged**